README.md

```diff
@@ -1,92 +1,60 @@
-```
-divisibleby
-```
-- [Filters](https://developers.hubspot.com/docs/cms/hubl/filters) are accessed with `|`. Ex: `|se` > Enter produces:
-```
-|selectattr("${attr}", ${exp_test})
-```
-- [Functions](https://developers.hubspot.com/docs/cms/hubl/functions) and [tags](https://developers.hubspot.com/docs/cms/hubl/tags) are accessed with `~`. Ex: `~hub` > Enter produces:
-```
-hubdb_table_rows(${table_id}, ${query})
-```
-- [HubL supported variables](https://developers.hubspot.com/docs/cms/hubl/variables) are accessed by typing the variable name alone. Ex: `content.ab` > Enter produces:
-```
-content.absolute_url
-```
-- [Module fields](https://developers.hubspot.com/docs/cms/building-blocks/module-theme-fields-overview) can be access by typing the field type (in JSON files only). Ex: `ri` > Enter produces:
-```
-{
-"name": "richtext_field",
-"label": "Rich text field",
-"required": false,
-"locked": false,
-"type": "richtext",
-"inline_help_text": "",
-"help_text": "",
-"default": null
-}
-```
-
-Other helpful snippets include:
-| Snippet Prefix | Description | Example |
-|-|-|-|
-| otrue | Generates overrideable=True for HubL tags | overrideable=True |
-| ofalse | Generates overrideable=False for HubL tags | overrideable=False |
-| for | Returns a basic if statement | {% for {item} in {dict} %} {{ item }} {% endfor %}" |
-| if | Returns a basic if statement | {% if {condition} %} do_something {% endif %} |
-| elif | Else if statement to be used within if statement | {% elif {condition} %} |
-| else | Else statement to be used within if statement | {% else %} |
-| hubldoc | Boilerplate html/HubL document |  |
-| hublblog | Boilerplate blog markup |  |
-
-
-
-### Beta features
-To enable beta features, naviate to `HubSpot > Beta` in your VS Code settings
-#### Current Beta features:
-- HubL inline error linting: Automatically checks your HTML + HubL and CSS + HubL files for HubL related errors
-### Common issues
-- If you are having trouble getting IntelliSense suggestions when in snippet placeholders you may need to add the following to your [User Settings](https://code.visualstudio.com/docs/getstarted/settings) `"editor.suggest.snippetsPreventQuickSuggestions": false`. If parameter suggestions are not showing up, set `"editor.parameterHints": true`.
-- To use Emmet within your `html-hubl` files, you can map `html-hub` to `html` in your settings under "Emmet: Include Languages":
-
-  ![image](https://user-images.githubusercontent.com/9009552/114593899-9e320500-9c5a-11eb-98c6-9de022344ebc.png)
-
-
- ### Contributing
- For more information about contributing, see the [contributing docs](./CONTRIBUTING.md).
+This extension is open source and we welcome contributions as well as issues for feature requests and bug reports. For more information about contributing, see the [contributing docs](https://github.com/HubSpot/hubspot-cms-vscode/blob/master/CONTRIBUTING.md) to get started.
```

Comment:
Can we add links to [report a bug](https://github.com/HubSpot/hubspot-cms-vscode/issues/new?assignees=&labels=bug&template=bug_report.md&title=) and [submit a feature request](https://github.com/HubSpot/hubspot-cms-vscode/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=)?